content/gocourse-2019-05-melbourne.slide

```diff
@@ -1150,6 +1150,6 @@ Dave Cheney Blog ([[https://dave.cheney.net/paste/the-past-present-and-future-of
 
 * Feedback
 
-Please take a moment to give anonymous [[https://docs.google.com/forms/d/e/1FAIpQLSeLKHWpq77ATHzgyVLMDsRAzuemJn-o9h4S8pMjdx8B9_IM4w/viewform][feedback]].
+Please take a moment to give anonymous feedback at [[http://bit.ly/gc-f6k]].
```

Comment:
The form this links to has a couple of bugs:

s/What went will/What went well/
s/Imporve/Improve/

Reply:
Also
s/helkpful/helpful/
s/apprecate/appreciate/

Reply:
also
s/imporve/improve/

(this one appears twice, once with lowercase, once with title case)

Reply:
wow. that is a new record. sorry.... 
thought i copied these from the previous feedback form, luckily i didn't ;)

Done.